portal/src/main/webapp/WEB-INF/jsp/patient_view/clinical_timeline.jsp

```diff
@@ -159,7 +159,7 @@
 
                 for (i=0; i < prefixes.length; i++) {
                     var prefix = prefixes[i];
-                    if (patientInfo[prefix+"OS_STATUS"] === "DECEASED" &&
+                    if (patientInfo[prefix+"OS_STATUS"] === "1:DECEASED" &&
```

Comment:
Are we not generaIizing for other survival data here?

Reply:
I think this file is also outdated. We are not using this anymore. I just want to keep the consistency between all the files, so I also changed this.